src/Components/GuessBlock/guessBlock.js

```diff
@@ -4,7 +4,7 @@ import { LetterBlock } from "../LetterBlock/letterBlock";
 
 export const GuessBlock = ({ word, data }) => {
   return (
-    <div className={data}>
+    <div className="guess">
```

Comment:
I forgot I had made this change, I don't know if you intended to the use the highly specific classes in some way but if so I'd still have the class be formatted with a generic "guess" that can be targetting for generic style a bit easier.
Something like this: `className={`guess ${data}`}`
But ideally, you don't want to use html classes in this way.

Try to give components as little information as possible, just the word in this case may be all that is needed.